source/meetings/general/2023-01-24.rst

```diff
@@ -0,0 +1,42 @@
+General Meeting 09:00 18-Sep-2023
```

Comment:
Should this be the date of the meeting (i.e. the 24th)? Or is it week-commencing?

Looking at other files would suggest the former…?

Reply:
Good catch. It should be the date. Fixing.

Reply:
Fixed here [25d0ae4](https://github.com/UKPythonAssociation/ukpa-internaldocs/pull/352/commits/25d0ae4d70a8cce4563ce3ca77cffd3ff41a0834)